lib/msal-node/tsconfig.json

```diff
@@ -1,30 +1,31 @@
 {
```

Comment:
Only relevant change here is the addition of  "resolveJsonModule": true. (I also fixed the indentation, and thus the big diff)